mlir/lib/Target/LLVMIR/ModuleTranslation.cpp

```diff
@@ -1029,11 +1029,17 @@ LogicalResult ModuleTranslation::convertGlobals() {
           debugTranslation->translateGlobalVariableExpression(op.getDbgExpr());
       llvm::DIGlobalVariable *diGlobalVar = diGlobalExpr->getVariable();
       var->addDebugInfo(diGlobalExpr);
+      // For fortran, the scope hierarchy can be
+      // variable -> module -> compile unit
```

Comment:
Modules are there in C++ also these days. So can we say "In languages like C++, Fortran etc variables ..."

Reply:
Or alternatively, 
-> remove the mention of Fortran. Just say that this hierarchy is allowed for languages supporting modules.
-> For Modula 2, Fortran like languages

Reply:
Thanks for the suggestion. I have updated the comments.